style.css

```diff
@@ -1,3 +1,61 @@
+:root {
+  --lightblue: #caf0f8;
+  --springblue: #ade8f4;
+  --darkblue: #03045e;
```

Comment:
Praise: Good use of variables
Suggestion: Name your variables based on how they will be used e.g. `--background` or `--episodecolor` this means if you decide to change the colour scheme in the future you won't need to change the variable names.